Makefile

```diff
@@ -5,6 +5,10 @@ build:
 format:
 	gofmt -s -w .
 
+imports:
+	go get -u golang.org/x/tools/cmd/goimports
+	goimports -local github.com/elastic/elastic-package/ -w .
+
```

Comment:
Should we just make this part of `format`?

Reply:
Fixed. Thanks!